static/index.html

```diff
@@ -30,6 +33,9 @@
 
     <!-- Things View -->
     <section data-view="things" class="selected" id="things-view">
+      <div class="title-bar">
+        <h1 class="things-header">Things</h1>
+      </div>
```

Comment:
After adding this div, the thing icons are no longer vertically centre aligned because the div is positioned 50% down within the remaining space.